### PR TITLE
Populate the S3 object key for the whitelist

### PR DESCRIPTION
**Pull request by emileswarts**
Our admin platform will populate this configuration file.
Eventually the Radius servers will read this file to populate the IP
whitelist.

### CODE REVIEW

File: govwifi-admin/cluster.tf [hunk 1]
```diff
@@ -88,6 +88,9 @@ resource "aws_ecs_task_definition" "admin-task" {
         },{
           "name": "S3_PUBLISHED_LOCATIONS_IPS_OBJECT_KEY",
           "value": "ips-and-locations.json"
+        },{
+          "name": "S3_WHITELIST_OBJECT_KEY",
```
Comment (kjdchapman) [inline govwifi-admin/cluster.tf]: Are both of the objects going in the same bucket? I don't know if I'm reading this change right.

Comment (emileswarts) [reply govwifi-admin/cluster.tf]: Yes the bucket was named govwifi-admin-staging or govwifi-admin-production

Comment (tommotaylor) [reply govwifi-admin/cluster.tf]: Same bucket different objects. Like saying "same folder, different files"

**Comment by tommotaylor** [comment]
Looks good to me

File: govwifi-admin/cluster.tf (hunk 1 above)
Comment (adrianclay) [reply govwifi-admin/cluster.tf]: Be nice if there were one bucket in London for the London frontends and one bucket in Dublin for the Dublin frontends.

Comment (emileswarts) [reply govwifi-admin/cluster.tf]: S3 Buckets are global.

Comment (emileswarts) [reply govwifi-admin/cluster.tf]: I stand corrected, ignore that

**Pull request merged**